components/mpas-seaice/src/column/ice_atmo.F90

```diff
@@ -318,8 +324,8 @@ subroutine atmo_boundary_layer (sfctype,            &
             tau = rhoa * rd * rd ! not the stress at zlvl
 
             ! high frequency momentum coupling following Roberts et al. (2014)
-            strx = tau * sqrt((uatm-uvel)**2 + (vatm-vvel)**2) * (uatm-uvel)
-            stry = tau * sqrt((uatm-uvel)**2 + (vatm-vvel)**2) * (vatm-vvel)
+            strx = tau * sqrt((uatm-uvel)**2 + (vatm-vvel)**2 + ugust**2 ) * (uatm-uvel)
+            stry = tau * sqrt((uatm-uvel)**2 + (vatm-vvel)**2 + ugust**2 ) * (vatm-vvel)
```

Comment:
Please note that these lines do not conserve and need to be changed.  If gustiness is to be included, it must be relative to the sea ice velocity.  It's a common misconception that sea ice travels always much more slowly than the wind the ice velocity differential need not be calculated.  However, this does not account for super-inertial coupling when sea ice can travel in the opposite direction of the wind during inertial loops, nor in high through-flow regions, such as Fram Strait, where sea ice frequently travels faster than the wind, or in low wind speed scenarios when internal ice and water stress are the predominant drivers.  In the current coding, it is possible for sea ice to travel at the same speed and direction as the wind, but still accelerate due to wind stress.